ver1

lastevil opened the pull request:
1 commit

Commit:
Add server netty, add new view of client app, make 2 module in 1 project. Create new dialog in client app.

Commit:
add file copy in local mode.
need add server msg

Commit:
add messaging... but failed to read class descriptor, I think this is java 11...

lastevil commented:
Еще в процессе вышла непонятная ошибка при передачи объектов... Возможно придется передавть другими спосабами...

vyrexd commented on cloud-server/src/main/java/gb/project/cloud/server/ServerApp.java:
```diff
@@ -0,0 +1,49 @@
+                        protected void initChannel(SocketChannel socketChannel) {
+                                socketChannel.pipeline().addLast(
+                                        new ObjectEncoder(),
+                                        new ObjectDecoder(ClassResolvers.weakCachingResolver(ClassResolvers.class.getClassLoader())),
```
Здесь как на уроке сделайте

Commit:
improve error:
java.io.InvalidClassException: failed to read class descriptor
Add:
- path server directory in message for clientAPP.
- navigating server folders
- download and upload files

Commit:
improve error:
Full screen resolution;
add min Height and Width window size

Commit:
123
